cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -433,7 +439,6 @@ if(CMAKE_HOST_WIN32)
   set(b2_cmd "b2")
   if("@MSVC@")
     set(HUNTER_MSVC_VCVARSALL "@HUNTER_MSVC_VCVARSALL@")
-    set(HUNTER_MSVC_ARCH "@HUNTER_MSVC_ARCH@")
```

Comment:
Why is the arch removed?

Reply:
This is removed as it is no longer used (see your other comment, I will try to write a good explaination of that on there).